package.json

```diff
@@ -56,6 +57,7 @@
     "postcss-cli": "8.3.1",
     "prettier": "2.2.1",
     "prop-types": "15.7.2",
+    "react-test-renderer": "17.0.2",
```

Comment:
This dep is required by the hooks testing library. See: https://react-query.tanstack.com/guides/testing.